xbmc/input/KeyboardStat.cpp

```diff
@@ -32,6 +32,8 @@
 #include "threads/SystemClock.h"
 #include "utils/log.h"
 
+#define HOLD_THRESHOLD 250
```

Comment:
That seem difficult, because you have to have a clear trigger between normal and long or it'll become messy.
What's the use case of having different durations?